storops/vnx/block_cli.py

```diff
@@ -871,6 +872,118 @@ def set_stats(self, enable=None):
                 cmd.append('-off')
         return cmd
 
+    @command
+    def get_ioclass(self, name=None):
+        cmd = ['nqm', '-ioclass', '-list']
+        cmd += text_var('-name', name)
+        return cmd
+
+    @command
+    def create_ioclass(
+            self, name, iotype='rw', lun_ids=None, smp_names=None,
+            ctrlmethod=VNXCtrlMethod.NO_CTRL, minsize=None, maxsize=None):
+        cmd = ['nqm', '-ioclass', '-create', '-name', name, '-iotype', iotype]
+        cmd += int_var('-minsize', minsize)
+        cmd += int_var('-maxsize', maxsize)
+        if lun_ids:
```

Comment:
lun_ids is a list of id, could not use `text_var`
As I found `lun_ids` must be a list of parameters, and cannot be joined as one single string parameter.